pkg/config/config.go

```diff
@@ -172,10 +172,12 @@ type Config struct {
 		} `mapstructure:"dynamodb"`
 
 		CosmosDB *struct {
-			Key       SecureString `mapstructure:"key"`
-			Endpoint  string       `mapstructure:"endpoint"`
-			Database  string       `mapstructure:"database"`
-			Container string       `mapstructure:"container"`
+			Key        SecureString `mapstructure:"key"`
+			Endpoint   string       `mapstructure:"endpoint"`
+			Database   string       `mapstructure:"database"`
+			Container  string       `mapstructure:"container"`
+			Throughput uint         `mapstructure:"throughput"`
+			Autoscale  bool         `mapstructure:"autoscale"`
```

Comment:
Do we line to add Automatic? as when we configure it we switch to automatic throughput.
Also there is a way to configure percentage as part of automatic

Reply:
Didn't quite understand the comment.
Was trying to add basic autoscale configuration as a start.
Once autoscale is enabled the container will scale from 10% of maximal value (throughput param) up to 100% as needed.
In the future if a requirement for increment percentage arises we can easily add it

Reply:
Got it (I was ref the use of `NewAutoscaleThroughputPropertiesWithIncrement`)